### PR TITLE
Improve OS migration script

### PR DESCRIPTION
**Pull request by njohner**
- We allow setting new permissions on existing repository folders. Note that we only do this if inheritance is blocked and local_roles are indicated in the Excel. These will replace the existing sharing permissions
- We skip the docproperties update
- We add commits during the migration, this is a test to see whether it will make the migration faster.
- I also improved logging a bit.

Tests were also updated, see https://github.com/4teamwork/opengever.core/pull/6792

For https://4teamwork.atlassian.net/browse/CA-1472

### CODE REVIEW

File: opengever/maintenance/scripts/repository_migration.py [hunk 1]
```diff
@@ -1013,6 +1013,7 @@ def create_repository_folders(self, items):
         self.start_bundle_import(tmpdirname)
 
         shutil.rmtree(tmpdirname)
+        transaction.commit()
```
Comment (lukasgraf) [inline opengever/maintenance/scripts/repository_migration.py:1016]: These should respect `options.dryrun`.

**Review by lukasgraf — CHANGES_REQUESTED** [review]
LGTM 👍 , except the `commit()` not respecting the dryrun flag.

File: opengever/maintenance/scripts/repository_migration.py (hunk 1 above)
Comment (njohner) [reply opengever/maintenance/scripts/repository_migration.py:1016]: True, thanks!

**Comment by njohner** [comment]
All done. As discussed, the last commit will be tested in the next test-migration, see if it really speeds the migration up.

**Review by lukasgraf — APPROVED** [review]
LGTM 👍

**Comment by njohner** [comment]
@lukasgraf can you have a look at the last commit? I had to exclude reindexing the SearchableText as we won't have the blobs during migration.

**Review by lukasgraf — APPROVED** [review]
LGTM 👍

**Pull request merged**